### PR TITLE
Kubernetes environment

### PR DESCRIPTION
**Pull request by tx19980520**
1. update code about patch
2. adjust to operator v0.1.12

### CODE REVIEW

**Comment by tx19980520** [comment]
Although now single local scheduler can run in the k8s cluster with the real golang function, we should test cross local scheduler call in the k8s cluster. @linxuyalun

**Review by linxuyalun — APPROVED** [review]
LGTM, I will add some comments and add a tiny revision in the later commit

**Comment by linxuyalun** [comment]
> Although now single local scheduler can run in the k8s cluster with the real golang function, we should test cross local scheduler call in the k8s cluster. @linxuyalun

We will do this in another PR

File: pkg/tools/k8sutils/k8sutils.go [hunk 1]
```diff
@@ -285,14 +301,14 @@ func GetWorkflowByName(name string) (*serverlessv1alpha1.Workflow, bool, error)
 		return nil, false, nil
 	}
 	var wf *serverlessv1alpha1.Workflow
-	switch obj.(type) {
+	switch obj := obj.(type) {
```
Comment (linxuyalun) [inline pkg/tools/k8sutils/k8sutils.go:304]: assigning the result of this type assertion to a variable could eliminate type assertions in switch cases (S1034)

**Pull request merged**